example/.gitignore

```diff
@@ -1,4 +1,4 @@
 node_modules/
 src/*.js
 lib
-yarn.lock
```

Comment:
This file was committed and gitignored. It should be either gitignored and rm'ed from the history or ungitignored and commited. I did the latter.